### PR TITLE
update tifi staking and add pool2

### PR DESCRIPTION
**Pull request by OscarYin**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/tifi-bank exports TVL: 
        
 
 
    bsc-staking               1.06 M
    bsc-pool2                 0
    bsc                       23.18 k
    staking                   1.06 M
    pool2                     0
    
    total                    23.18 k

**Comment by llamatester** [comment]
Error while running adapter at projects/tifi-bank: 
        
 
  
    
    
    TypeError: t.toLowerCase is not a function
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/unwrapLPs.js:727:56
        at Array.map (<anonymous>)
        at sumTokens2 (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/unwrapLPs.js:727:37)
        at sumUnknownTokens (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/cache/sumUnknownTokens.js:379:9)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/unknownTokens.js:98:35
        at getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:29)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:148:15
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:136:29
        at Array.map (<anonymous>)

**Comment by llamatester** [comment]
The adapter at projects/tifi-bank exports TVL: 
        
 
 
    bsc-staking               1.06 M
    bsc                       23.18 k
    bsc-pool2                 946.10 k
    staking                   1.06 M
    pool2                     946.10 k
    
    total                    23.18 k

**Comment by OscarYin** [comment]
Perfect, I think the adaptors' result is exactly what we want to show for the TiFi, please help merge this, thank you!

File: projects/tifi-bank/index.js [hunk 1]
```diff
@@ -1,9 +1,35 @@
...
+  "0xB62BB233Af2F83028Be19626256A9894B68AAe5E"
+]
+
 
 module.exports = {
   bsc: {
     tvl: getUniTVL({ chain: 'bsc', factory: '0xb3456550c17128ca7ebbcc47d4be6cae29d43853', }),
-    staking: staking('0x0AEfF3d761F6706295f3828C87ccE29c9418a93B', '0x17E65E6b9B166Fb8e7c59432F0db126711246BC0', 'bsc')
+    staking: stakings(stakingContracts, '0x17E65E6b9B166Fb8e7c59432F0db126711246BC0', 'bsc'),
+    pool2: sumTokensExport({ tokens: lpTokens, owners: lpContract, useDefaultCoreAssets: true }),
```
Comment (g1nt0ki) [inline projects/tifi-bank/index.js:33]: sorry, I dont think tokens locked in pinksale can be considered your pool2, it needs to be LPs deposited by users in contracts controlled by you.

Comment (OscarYin) [reply projects/tifi-bank/index.js:33]: @g1nt0ki Even those locks are all created by us?

Comment (OscarYin) [reply projects/tifi-bank/index.js:33]: We are using the pink lock to add liquidity for the pool we use in our Dapp and show our users the numbers and dates we locked.

Comment (OscarYin) [reply projects/tifi-bank/index.js:33]: And here are our lock records on the pink sale, you can see the token numbers are exactly the same means the locks is all created by us.

https://www.pinksale.finance/pinklock/detail/0x707B6F02fFC0C7fD9fe3a4F392Aef47218021337?chain=BSC
https://www.pinksale.finance/pinklock/detail/0xB62BB233Af2F83028Be19626256A9894B68AAe5E?chain=BSC

Comment (g1nt0ki) [reply projects/tifi-bank/index.js:33]: > @g1nt0ki Even those locks are all created by us?

*sorry, no, we have not done this for other projects. LP tokens locked by team shows commitment to the project but cant be considered pool2 https://docs.llama.fi/list-your-project/what-to-include-as-tvl

**Comment by llamatester** [comment]
The adapter at projects/tifi-bank exports TVL: 
        
 
 
    bsc-staking               992.58 k
    bsc                       23.09 k
    staking                   992.58 k
    
    total                    23.09 k

File: projects/tifi-bank/index.js (hunk 1 above)
Comment (OscarYin) [reply projects/tifi-bank/index.js]: @g1nt0ki got it, I just commit the code to remove the pool2, please help us merge the PR, thank you!

**Pull request merged**